tools/openms/MapAlignerPoseClustering.xml

```diff
@@ -1,19 +1,43 @@
-<tool id="MapAlignerPoseClustering" name="MapAlignerPoseClustering" version="2.2.0">
+<?xml version='1.0' encoding='UTF-8'?>
+<!--This is a configuration file for the integration of a tools into Galaxy (https://galaxyproject.org/). This file was automatically generated using CTDConverter.-->
+<!--Proposed Tool Section: [Map Alignment]-->
+<tool id="MapAlignerPoseClustering" name="MapAlignerPoseClustering" version="2.3.0">
```

Comment:
Great work @blankclemens! This is the only tool which I think might make problems.
Following the documentation I think this tool can take N files and will create N files and not one. http://ftp.mi.fu-berlin.de/pub/OpenMS/release-documentation/html/TOPP_MapAlignerPoseClustering.html

I made these changes here for the old version: https://github.com/galaxyproteomics/tools-galaxyp/pull/254/files
If this makes sense to you, lets merge the older one and then rebase this one.